verify-matching-service-test-tool/src/main/java/uk/gov/ida/verifymatchingservicetesttool/configurations/ApplicationConfiguration.java

```diff
@@ -32,6 +48,51 @@ public URI getLocalMatchingServiceAccountCreationUrl() {
     }
 
     public Boolean getLocalMatchingServiceUsesUniversalDataSet() {
-        return localMatchingService.getUsesUniversalDataset();
+        return localMatchingService.usesUniversalDataset();
+    }
+
+    public FilesLocator getFilesLocator() {
+        return filesLocator;
+    }
+
+    public JsonValidator getJsonValidator() {
+        return jsonValidator;
+    }
+
+    public FileUtils getFileUtils() {
+        return fileUtils;
+    }
+
+    private String getDatasetType() {
+        return localMatchingService.usesUniversalDataset() ? "universal-dataset" : "legacy";
+    }
+
+    private String getExamplesFolder(String examplesFolderLocation) {
+        if (examplesFolderLocation != null) {
+            String exampleScenariosFolderLocation = getAbsoluteFilePath(examplesFolderLocation);
+            if (exampleScenariosFolderLocation != null &&
+                    checkIfFileDoesNotExist(exampleScenariosFolderLocation, FolderName.MATCH_FOLDER_NAME) &&
+                    checkIfFileDoesNotExist(exampleScenariosFolderLocation, FolderName.NO_MATCH_FOLDER_NAME)) {
+                System.out.println(String.format("%sTest Run Failed%s", Color.RED, Color.NONE));
+                System.out.println(String.format("%sEnsure the specified example scenarios folder '%s' " +
+                        "has 'match' and 'no-match' sub-folders%s", Color.YELLOW, exampleScenariosFolderLocation, Color.NONE));
+                System.exit(2);
```

Comment:
I have a different concern to Codacy here...

Only the main method should call ```System.exit()```, the config class definitely **shouldn't**. 😱 

I would suggest this method to throw an exception so that it bubbles up to the main method and then it falls over there.